doc/syntax.rst

```diff
@@ -47,28 +47,28 @@ the following:
 
 * Case sensitivity.
 
-.. 
```

Comment:
Ah, the reason for this erroneous space is that my text editor (sublime text) has issues with ReST syntax highlighting if its missing (since it thinks the `..` is the beginning of a directive 😆)